manager_contacts/src/ContactsManager.java

```diff
@@ -0,0 +1,22 @@
+public class ContactsManager {
+    Contact [] myFriends;
+    int friendsCount;
+
+    ContactsManager(){
+        this.friendsCount = 0;
+        this.myFriends = new Contact[500];
+    }
+
+    void addContact(Contact contact){
+        myFriends[friendsCount] = contact;
+        friendsCount++;
+    }
+
+    void searchContact(String searchName){
+        for(int i=0; i<friendsCount; i++){
+            if(myFriends[i].name.toLowerCase().contains(searchName.toLowerCase())){
+                System.out.println(myFriends[i].name +" "+myFriends[i].phoneNumber+" "+myFriends[i].email);
```

Comment:
@valetodo Instead of print the contact found, you should return it. The responsibility of display the contact is not part of this class, this is from the Console or the UI that the class will have